routing/router.go

```diff
@@ -366,8 +366,10 @@ func (a alice) then(h http.Handler) http.Handler {
 func buildHttpHandlerLogger(route Route, handlerFunction http.Handler) http.Handler {
 	c := alice{}
 	c = c.append(hlog.NewHandler(zerolog.With().Str("route", route.Name).Logger()))
-	c = c.append(hlog.HostHandler("Host", true))
-	c = c.append(hlog.RequestIDHandler("RequestID", "Request-Id"))
+	c = c.append(hlog.HostHandler("host", true))
+	c = c.append(hlog.RemoteAddrHandler("remoteAddr"))
+	c = c.append(hlog.CustomHeaderHandler("xRequestId", "X-Request-Id"))
+	c = c.append(hlog.RequestIDHandler("kialiRequestId", ""))
```

Comment:
Can we keep this as `RequestID` for now and maybe later we can combine the two request ids?

Reply:
see https://github.com/kiali/kiali/pull/8411/commits/7b96e17e92365351a200a8e0635f31a0fee34c94

Reply:
and finally https://github.com/kiali/kiali/pull/8411/commits/914da7e92649ee8acfaa483ca1e1505f68ed13b3

Reply:
Gonna look like this now:
```json
{
  "level": "trace",
  "route": "GraphNamespaces",
  "host": "localhost",
  "remoteAddr": "127.0.0.1:37090",
  "xRequestId": "d0h2olpa02tc739ueei0",
  "group": "graph",
  "ts": "2025-05-12T17:13:59Z",
  "msg": "Generating config for [common] graph..."
}
```

Reply:
(if there was a true X-Request-ID header, the json keys would be the same, you'd just see the "real" X-Request-Id value as the xRequestId value)